style.css

```diff
@@ -32,29 +28,107 @@ body {
   font-size: 18px;
   line-height: 24px;
   color: #fff;
+  padding: 2%;
+}
+
+@media screen and (min-width: 375px) and (max-width: 768px) {
+  .menu-items.show {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start space-around;
+    align-items: start;
+    background-color: #3c3a39;
+    height: 100%;
+    width: 100%;
+    transition: 0.3s;
+    position: fixed;
+    top: 0;
+    left: 0;
+    list-style: none;
+    margin: 0;
+    padding-top: 15%;
+    padding-right: 10%;
+  }
+
+  .menu-items.show hr {
+    min-width: 279px;
+    height: 1px;
+    background: #6f6c6b;
+    padding-right: 24px;
+  }
+
+  .menu-items.show .email {
+    display: none;
+  }
+
+  .menu-items.show .nav-link a {
+    text-decoration: none;
+    color: #fff;
```

Comment:
- [ ] As part of the requirement for this milestone is to try and replicate the Figma designs as much as you can, looking at the lines in-between the menu items, you can see that yours is ticker and white, which is different from the Figma design, I'll suggest you get the correct thickness and color from the Figma design and update yours. :+1: :+1: 

- - Your design:
![Selection_007](https://github.com/allenkays/Portfolio-setup-and-mobile-first/assets/107632780/ad63f528-9305-45dc-83ca-71672e126160)

 - - Figma design:
![Screenshot from 2023-08-01 19-50-44](https://github.com/allenkays/Portfolio-setup-and-mobile-first/assets/107632780/e438f1cb-9a65-43d5-8c3d-9274a25bfc34)